docs/external plugins/Verifier/vulnerabilityreport.md

```diff
@@ -0,0 +1,333 @@
+
+### CLI
+
+Sample JSON
+```json
+{
+    "store": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "oras",
+            }
+        ]
+    },
+    "policy": {
+        "version": "1.0.0",
+        "plugin": {
+            "name": "configPolicy"
+        }
+    },
+    "verifier": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "vulnerabilityreport",
+                "artifactTypes": "application/sarif+json",
+                "maximumAge": "24h",
+                "disallowedSeverity": ["high", "critical"],
+                "denylistCVEs": ["CVE-2021-44228"]
+            }
+        ]
+    }
+}
+```
+
+## Advanced Scenario
+
+Alice has a Kubernetes cluster. She wants to enable vulnerability report verification. In particular, she wants to make sure all container images used in K8s resources have at least one valid vulnerability report attached to the image. The report must:
+- be in SARIF format
+- generated from Trivy or Grype
+- created less than 24 hours ago
+- has no HIGH or CRITICAL severity level vulnerabilities in the report
+- does not contain the log4shell CVE 
+
+Furthermore, each vulnerability report must have a valid Notary Project signature attached to it.
+
+First, follow only steps 1 and 2 of the [manual quickstart](../../quickstarts/quickstart-manual.md). 
+
+Next, install the vulnerability report constraint template and constraint. The Constraint Template defines the policy "all container images used in K8s resources have at least one valid vulnerability report attached to the image and the report has a valid Notary Project signature"
+
+```shell
+kubectl apply -f https://deislabs.github.io/ratify/library/vulnerability-report-validation/template.yaml
+
+kubectl apply -f https://deislabs.github.io/ratify/library/vulnerability-report-validation/samples/constraint.yaml
+```
+
+An image `myregistry.io/vuln/alpine:3.18.2` is scanned and a vulnerability report is generated. A reference artifact is generated:
+1. Use Trivy to scan `myregistry.io/vuln/alpine:3.18.2` and output a report in SARIF format
+    ```shell
+    trivy image -q -f sarif myregistry.io/vuln/alpine:3.18.2 > trivy-sarif.json
+    ```
+2. A tool such as `oras` is used to package, attach, and then push the report to registry
+    - `artifact-type` MUST be `application/sarif+json`
+    - file media type (layer) MUST be `application/sarif+json`
+    - `org.opencontainers.image.created` annotation with RFC3339 formatted current timestamp
+    ```shell
+    oras attach \
+        --artifact-type application/sarif+json \
+        --annotation "org.opencontainers.image.created=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" \
+        myregistry.io/vuln/alpine:3.18.2 \
+        trivy-sarif.json:application/sarif+json
+    ```
+3. Use `notation` to sign the report
```

Comment:
should we make notation a link? jsut in case customer is not familiar with the tool, we can also add link for docs on how to generate key etc